phpBB/includes/functions_content.php

```diff
@@ -1248,6 +1236,18 @@ function parse_attachments($forum_id, &$message, &$attachments, &$update_count,
 
 	$unset_tpl = array_unique($unset_tpl);
 
+	// Sort correctly
+	if ($config['display_order'])
+	{
+		// Ascending sort
```

Comment:
This is the same as under the acp settings for the display order. Why do you think it should be the other way round?